hack/local-up-cluster.sh

```diff
@@ -187,6 +187,7 @@ LOG_LEVEL=${LOG_LEVEL:-3}
 CONTAINER_RUNTIME=${CONTAINER_RUNTIME:-"docker"}
 CONTAINER_RUNTIME_ENDPOINT=${CONTAINER_RUNTIME_ENDPOINT:-""}
 IMAGE_SERVICE_ENDPOINT=${IMAGE_SERVICE_ENDPOINT:-""}
+ENABLE_CRI=${ENABLE_CRI:-"true"}
```

Comment:
@deads2k the "default" of false accidentally snuck in with my PR here: https://github.com/kubernetes/kubernetes/commit/bd912f50bae3855082bdcc8368b8a85eefe1e8e8#diff-ed3df710e9af7cd30a185896a60897d9R578. It was not supposed to.